pkg/model/helper.go

```diff
@@ -29,6 +29,9 @@ type Helper struct {
 	// A map of operation type and resource name to
 	// aws-sdk-go/private/model/api.Operation structs
 	opMap *OperationMap
+	// Instructions to the code generator how to handle the API and its
+	// resources
+	generatorConfig *GeneratorConfig
```

Comment:
I am guessing this is pathway to adding more customizations to code-generators. Super Cool Stuff! 💯